### DISCUSSION TITLE
SMI timeouts

### DISCUSSION THREAD

**Discussion by rush0815** [cariboulabs/cariboulite#138] [General]
Hi there,
there are some issues open according the smi read timeout running SoapySDRServer. But unfortunately no one has replied there yet.
I have installed cariboulite as shown in the README. All seems to be working, but no samples at all :-/ Only timeouts.

Did anyone from you got around that ?

Kind regards, rush

**Comment by tpresence** [comment]
I have this on my pi4 as well.  SoapySDRServer binds properly, and cubicsdr will connect, but when I try to attach to HIF or S1G, all read errors.

**Comment by K7MDL2** [comment]
It is worth trying the latest build as of Dec 7, 2023.  The install.sh script, with one mod on line 144 to add sudo in front of make, it has been working well, CubicSDR has been working pretty well, and some other SDR apps that talk SoapyDR sort of work here.  Progress continues.  Pi4B with SSD and bullseye, 64 bit.
